cpp/ql/src/experimental/Security/CWE/CWE-788/AccessOfMemoryLocationAfterEndOfBufferUsingStrncat.c

```diff
@@ -0,0 +1,4 @@
+ 
+strncat(dest, src, sizeof(dest) - strlen(dest)); //bad:
```

Comment:
We tend to capitalize these `BAD` and `GOOD` annotations in examples, and generally, there should be an explanation for why they're annotated as such.
```suggestion
strncat(dest, source, sizeof(dest) - strlen(dest)); // BAD: writes a zero byte past the `dest` buffer.
```